### DISCUSSION TITLE
Get information about NFT in Wallet

### DISCUSSION THREAD

**Discussion by FloSuckfuell** [Web3Auth/Web3Auth#1178] [Help]
How can I get the information about an NFT inside a Web3Auth Wallet? 
I know how to get the wallet address but not how to see if a certain NFT is inside. I also didn't find something regarding this in the docs. 
Could anybody help? 🙏🏼
Is there an API I can use?

**Comment by FloSuckfuell** [comment]
We get no answer for the Opensea API. 
Can I use the wallet for logging on to Opensea? Every time I use the mail I generated the torus wallet with, it generates a new one for opensea, or torus.

**Comment by shahbaz17** [comment]
Hey @FloSuckfuell 

Is this with Torus Wallet? If not, Which SDK of Web3Auth you used?

**Reply by FloSuckfuell** [reply]
Its plug and play modal SDK https://web3auth.io/docs/sdk/web#web3auth-plug-and-play-modal-sdk

**Reply by Saviour1001** [reply]
Web3Auth SDK essentially helps out with wallet creation. You can check the NFTs of a particular wallet using APIs from other analytics tools like [Moralis](https://docs.moralis.io/web3-data-api/evm/nft-api) etc.

**Reply by animusha** [reply]
@shahbaz17 @Saviour1001 
Here are the steps that were followed - 

I used the credentials to generate access token -
https://docs.venly.io/api/authentication/authenticate

Then I tried to use this token to get nfts on wallet address using the following api -
https://docs.venly.io/api/api-products/wallet-api/retrieve-non-fungible-tokens

then we pass the wallet address and it gives 401 unauthorized.
